packages/components/src/Badge/Badge.module.css

```diff
@@ -0,0 +1,105 @@
+.badge {
+  --dt-color-background-default: var(--color-gray-300);
```

Comment:
Looking at the code below instead of creating a variable that gets used only once I recon we update this so we set this on `.badge` class and then update that value in the modifier classes like `active`.

Also its good opportunity to use a default value in your CSS variable instead of having to declare it at the top of the stylesheet.

CSS variable is pretty handy that we can do something like:

```
.badge {
    <!-- other properties -->
    background-color: var(--badge-background-color, var(--color-gray-300))
}
```

Then later in the stylesheet, we just reassign the CSS variable as usual in the modifier class, ie:

```
.active {
    --badge-background-color: var(--color-blue-500)
}
```